navi/src/test/java/com/trello/navi/NaviFragmentTest.java

```diff
@@ -295,4 +295,9 @@ public void restoreInstanceStatePersistableListener() {
     exception.expect(IllegalArgumentException.class);
     emitter.addListener(Event.DETACHED_FROM_WINDOW, mock(Listener.class));
   }
+
+  @Test public void onViewCreatedListener() {
```

Comment:
Additionally, there should be this test in `NaviActivityTest` to verify that it doesn't have an `onViewCreated()` method.